### ISSUE TITLE
Dynamically Update GeoJSON Options e.g. Color 

### ISSUE THREAD

**Issue by Robbie-Palmer** [thedirtyfew/dash-leaflet#42]
I would like to be able to set the color of the geometries inside a GeoJSON layer from a callback
I have tried doing this by setting the output of a callback to be the options property on the layer but it doesn't update

Minimal reproducible example below, trying to change the colour of a triangle via radio buttons:
```
import dash
import dash_html_components as html
import dash_leaflet as dl
from dash.dependencies import Output, Input
from dash_bootstrap_components import RadioItems

triangle = dict(type='Polygon',
                coordinates=(((-104, 39),
                              (-104, 45),
                              (-90, 39),
                              (-104, 39)),))
geojson_layer = dl.GeoJSON(id='geojson_layer', data=dict(type='FeatureCollection', features=[triangle]))
map = dl.Map([dl.TileLayer(), geojson_layer], center=[39, -98], zoom=4, style=dict(height='50vh'), id="map")

geojson_colours = [dict(label='Green', value='#32a852'),
                   dict(label='Blue', value='#3268a8'),
                   dict(label='Red', value='#cf241b')]
colour_radio = RadioItems(id='colour-radio',
                          options=geojson_colours,
                          value=geojson_colours[0]['value'],
                          className='custom-radio')

app = dash.Dash()
app.layout = html.Div([map, colour_radio])


@app.callback(Output(geojson_layer.id, 'options'), [Input(colour_radio.id, 'value')])
def geojson_colour(colour):
    return dict(color=colour)


if __name__ == '__main__':
    app.run_server()
```

**Comment by emilhe** [comment]
Ah, yes, that's actually a bug. Thanks for the great example - it made it easy to find the issue. This build should fix the issue,

https://pypi.org/project/dash-leaflet/0.1.2rc4/

**Comment by Robbie-Palmer** [comment]
Thanks @emilhe!
I have this working now, but I did run into an other hurdle
My actual use case is to update both the data and change the colour based on a single item of user input
This doesn't work from a single callback
Here's the above callback adapted to reproduce this:
```
@app.callback([Output(geojson_layer.id, 'data'), Output(geojson_layer.id, 'options')],
              [Input(colour_radio.id, 'value')])
def geojson_colour(colour):
    return dict(type='FeatureCollection', features=[triangle]), dict(color=colour)
```
But I can functionally get what I need by chaining two callbacks together
```
@app.callback(Output(geojson_layer.id, 'data'), [Input(colour_radio.id, 'value')])
def geojson_data(colour):
    return dict(type='FeatureCollection', features=[triangle])


@app.callback(Output(geojson_layer.id, 'options'),
              [Input(colour_radio.id, 'value'), Input(geojson_layer.id, 'data')])
def geojson_colour(colour, geojson):
    return dict(color=colour)
```

**Comment by emilhe** [comment]
Ah, yes, i see. I just made a few more tweaks to support that usecase. It should work in the latest build,

https://pypi.org/project/dash-leaflet/0.1.2rc5/

\emher

**Comment by Robbie-Palmer** [comment]
Great thank-you! That's working now :)

**Comment by emilhe** [comment]
Great! I have now released a version 0.1.3 which should work :)
